account_financial_risk/wizards/partner_risk_exceeded_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2016-2018 Tecnativa - Carlos Dauden
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="partner_risk_exceeded_wizard" model="ir.ui.view">
+        <field name="name">Partner risk exceeded</field>
+        <field name="model">partner.risk.exceeded.wiz</field>
+        <field name="arch" type="xml">
+            <form string="Risk exceeded">
+                <p>The partner has exceeded his risk</p>
+                <field name="exception_msg" colspan="2" nolabel="1" />
+                <group>
+                    <field name="partner_id" />
+                </group>
+                <footer>
+                    <button
+                        string="Continue"
+                        class="oe_highlight"
```

Comment:
Nitpick: `oe_highlight` deprecated in favor of `btn-primary` [[source](https://github.com/odoo/odoo/blob/4a8239c86e3fdf2830bd4f52d2424fe0f83d4639/addons/web/static/src/js/core/dom.js#L404)]
```suggestion
                        class="btn-primary"
```